app/src/main/java/com/example/keybindhelper/ui/Projects/ProjectsFragment.kt

```diff
@@ -39,4 +42,8 @@ class ProjectsFragment : Fragment() {
         super.onDestroyView()
         _binding = null
     }
-}
\ No newline at end of file
+
+    fun openKeybindFragment() {
+        TODO("Not yet implemented")
+    }
```

Comment:
This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.